pkg/controller/networkpolicy/reconciler.go

```diff
@@ -71,6 +75,23 @@ type RuntimeNetworkConfig struct {
 	BlockCIDRs []string
 }
 
+// GetBlockedNetworkPeers returns a list of CIDRs to exclude from a NetworkPolicy IPBlock. ipFamily should match the
+// IP family of the IPBlock.CIDR value. The resulting list still needs to be filtered for subsets of IPBlock.CIDR.
+func (r RuntimeNetworkConfig) GetBlockedNetworkPeers(ipFamily gardencore.IPFamily) []string {
+	// NB: BlockCIDRs can contain both IPv4 and IPv6 CIDRs.
+	var peers = append([]string(nil), r.BlockCIDRs...)
+
+	if len(r.IPFamilies) == 1 && r.IPFamilies[0] == ipFamily {
+		peers = append(peers, r.Pods, r.Services)
```

Comment:
Don't we need to block seed ranges as well?

Reply:
@axel7born I really don't know. Can somebody please check it?

Reply:
I'm not sure, what the original intention of this networkpolicy was. But my question should not block this PR.